extensions/wikia/FandomCreator/FandomCreatorApi.php

```diff
@@ -14,10 +16,32 @@ class FandomCreatorApi {
 	/**
 	 * FandomCreatorApi constructor.
 	 * @param string $baseUrl
-	 * @param $communityId
+	 * @param string $communityId
 	 */
-	public function __construct($baseUrl, $communityId) {
+	public function __construct(string $baseUrl, string $communityId) {
 		$this->baseUrl = $baseUrl;
 		$this->communityId = $communityId;
 	}
-}
\ No newline at end of file
+
+	public function getCommunity() {
+		$response = $this->doApiRequest("{$this->baseUrl}/communities/{$this->communityId}");
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this unused "$response" local variable. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1481)